app/controllers/planning_applications_controller.rb

```diff
@@ -144,6 +145,17 @@ def view_recommendation
     @recommended_date = @planning_application.recommendations.last.created_at.strftime("%d %b %Y")
   end
 
+  def save_assessment
+    @planning_application.public_comment = params[:planning_application][:public_comment]
+    recommendation = @planning_application.recommendations.build(assessor: current_user)
+    recommendation.assessor_comment = params[:recommendation][:assessor_comment]
+    recommendation.save(validate: false)
```

Comment:
why is this needed ?

Reply:
A recommendation is made of these fields - the public_comment on the form is associated with the planning application whereas the recommendation has these separate params. I'm using a separate action to save the recommendation because we want validate: false - its an AC on the ticket to not enforce validations when saving as draft